### PR TITLE
Fix packaging 2

### PR DESCRIPTION
**Pull request by tomdee**

### CODE REVIEW

**Comment by tomdee** [comment]
@caseydavenport Assuming the tests pass, this should be ready for you to take a look at.

**Comment by caseydavenport** [comment]
Tests are still failing.

**Comment by tomdee** [comment]
@caseydavenport Yep, I forgot to update the circle CI target.

File: calico_cni/nose.cfg [hunk 1]
```diff
@@ -1,4 +0,0 @@
-[nosetests]
```
Comment (caseydavenport) [inline calico_cni/nose.cfg]: The make commands still reference this file but it's gone now.

**Comment by caseydavenport** [comment]
One question, but LGTM.

**Comment by tomdee** [comment]
I'd forgotten to re-add it. Now done so merging.

**Pull request merged**